### DISCUSSION TITLE
10주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#30] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by alanhakhyeonsong** [comment]
병렬 스트림을 한분이라도 궁금해하실 줄 알았지만, 그냥 설명해봤습니다... 도움이 되셨길^^;

**Reply by alanhakhyeonsong** [reply]
오늘 말했던 것들...

- [포크/조인 프레임워크란 무엇인가?](https://devbksheen.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%ED%8F%AC%ED%81%AC%EC%A1%B0%EC%9D%B8-%ED%94%84%EB%A0%88%EC%9E%84%EC%9B%8C%ED%81%AC%EB%9E%80-%EB%AC%B4%EC%97%87%EC%9D%B8%EA%B0%80)
- [parallelStream 완전분석](https://n1tjrgns.tistory.com/292)

**Comment by opgumo3** [comment]
2주에 걸쳐 '람다와 스트림' 챕터를 읽고 생각해보니, 그동안 스트림을 스트림답지 않게 사용했던 것 같습니다~..
더 사용해서 익숙해지는 것부터 해야 할 것 같습니다..

**Comment by sdh46** [comment]
3연속 발표... 부족한 발표 듣느라 고생하셨습니다. 그리고 이번엔 병렬 스트림에 대해서 궁금증을 가지지 않고 그대로 넘어갔네요... 좀 더 궁금증을 가지는 연습을 해보아야겠습니다.

**Reply by narangd0** [reply]
갈 수록 점점 느신 것 같아요 굿 🤣

**Reply by sdh46** [reply]
감사합니다!

**Comment by narangd0** [comment]
병렬화, 중개 어댑터 같은 개념이 생소해서 조금 어려웠던 것 같다.
써보지 않은 스트림 메소드들도 있고, 잘쓰던 것들도 있었는데 한 번 배우고 나니까 좋은 것 같다...

**Comment by JeongJaew0n** [comment]
스트림 하면서 함수형 처리에 대한 이해가 늘어서 좋았습니다.
@alanhakhyeonsong 병렬 처리 설명 좋았습니다 👍

**Comment by SuJeong68** [comment]
해당 챕터를 읽은 이후 stream을 더 자주 사용하고, 더 주의하게 되는거 같아요. parallel 메서드 동작방식에 대해 왜 궁금해 하지 않았는지 스스로도 이해하지 못했습니다. 더 깊이 생각해보도록 다시 마음을 다잡아야겠어요!

**Comment by Kurmarkpzgr** [comment]
함수형을 사용해본지 오래되었었다 보니 평소보다 이해에 좀 더 시간이 걸렷던거 같습니다. 
써보고 싶내요...
